lib/ReactTags.js

```diff
@@ -224,6 +225,7 @@ class ReactTags extends React.Component {
 ReactTags.defaultProps = {
   tags: [],
   placeholder: 'Add new tag',
+  noSuggestionsMessage: 'No suggestions found',
```

Comment:
I think the default should be an empty string or `null` to make this feature opt-in and avoid changing the existing behaviour.

Reply:
I will set it to null